internal/cmd/marketplace/__snapshots__/apply_test.snap

```diff
@@ -0,0 +1,40 @@
+
+[TestApplyPrintApplyOutcome/should_match_snapshot_with_both_valid_files_and_validation_errors - 1]
+3 of 4 items have been successfully applied:
+
+  ITEM ID  NAME         STATUS    
+
+  id1      some name 1  Updated   
+  id2      some name 2  Inserted  
+  id3      some name 3  Inserted  
+
+1 of 4 items have not been applied due to validation errors:
+
+  ITEM ID  NAME         VALIDATION ERRORS      
+
+  id4      some name 4  some validation error  
+
+---
+
+[TestApplyPrintApplyOutcome/should_match_snapshot_with_validation_errors_only - 1]
+2 of 2 items have not been applied due to validation errors:
+
+  ITEM ID  NAME         VALIDATION ERRORS               
+
+  id3      some name 3  some validation error           
+  id4      some name 4  some other validation error     
+                        and also some other validation  
+                        error                           
```

Comment:
Might it be better to expand the 'VALIDATION ERRORS' column to improve readability?

Reply:
I'll see what I can do with the tables library

Reply:
I'd give a shot to : `table.SetAutoWrapText(false)`

Reply:
yep seems good, done!